content/applications/marketing/marketing_automation/getting_started/first_campaign.rst

```diff
@@ -1,70 +1,113 @@
 
-- :guilabel:`Templates`: represents the number of pre-configured mail templates being used in this
-  particular campaign. (Templates can always be created on-the-fly, as well).
-- :guilabel:`SMS`: represents the number of personalized SMS messages connected to this campaign.
-- :guilabel:`Clicks`: represents the number of times attached links have been clicked by recipients
-  of this campaign.
-- :guilabel:`Participants`: represents the number of contacts that have directly participated in
-  this campaign.
+A target audience can be configured by entering specific criteria for Odoo to use when determining
+to whom this marketing automation campaign should be sent. Creating specific target audiences can
+help businesses deliver tailored messages and campaigns that appeal to a niche demographic, thus
+providing more value and personalization to all subsequent communications.
 
-**Fields**
+In the :guilabel:`Target` field, use the drop-down menu to choose which model the target audience
+filters should be based on (e.g. :guilabel:`Contact`, :guilabel:`Lead/Opportunity`,
+:guilabel:`Sales Order`, etc.).
 
-- :guilabel:`Name`: represents the name of the marketing automation campaign being created.
-- :guilabel:`Target`: this field is a drop-down menu to choose which model is targeted by this
-  campaign (i.e., by Contacts, Sales Order, Lead/Opportunity, etc.).
-- :guilabel:`Filter`: this field provides numerous configurable criteria that can be used to further
-  narrow down the target recipients/audience for the marketing automation campaign.
+Select :guilabel:`Search More...` from the drop-down menu to reveal a :guilabel:`Search: Target`
+pop-up window containing all 157 target options.
 
-Campaign filters
-================
+Once a :guilabel:`Target` is selected, there's a :guilabel:`Unicity based on` field. This field is
+used to avoid duplicates based on the model chosen in the :guilabel:`Target` field.
 
-To add a :guilabel:`Filter` to the target audience, click :guilabel:`Add Filter`, and a node field
-appears. In the node field, a custom equation can be configured for Odoo to use when filtering who
-to include (and exclude) in this specific marketing campaign.
+.. example::
+   If :guilabel:`Customers` is chosen as the :guilabel:`Target`, select :guilabel:`Email` in the
+   :guilabel:`Unicity based on` field so Odoo doesn't process any records that have the same Email
+   address.
 
-.. image:: first_campaign/filter-node.png
-   :align: center
-   :alt: A filter node in Odoo Marketing Automation.
+Select :guilabel:`Search More...` from the :guilabel:`Unicity based on` drop-down menu to reveal all
+129 options in a pop-up window.
+
+Lastly on the campaign detail form is the :guilabel:`Filter` field. This is where a more specific
+audience filter can be configured to add specificity to the recipients of the marketing automation
+campaign.
+
+If left alone, the :guilabel:`Filter` field reads: :guilabel:`Match **all records**`. That means
```

Comment:
Apparently asterisks don't work inside of a guilabel, they just show up as asterisks :)
```suggestion
If left alone, the :guilabel:`Filter` field reads: :guilabel:`Match all records`. That means
```